### ISSUE TITLE
Get Text Records does not work

### ISSUE THREAD

**Issue by vinny-sabatini** [bluecatlabs/gateway-workflows#120]
The `/configurations/{configuration}/views/{view}/zones/{zone}/text_records/` API call does not work.

I have attempted to fix this by following the get CNames code, however when I make the get text records API call with the same parameters as the get CName records call,  I get a 404 return code with the response `No matching Zone(s) found`. 

Here is the WIP PR: https://github.com/bluecatlabs/gateway-workflows/pull/119

Here is the block of code that does not appear to be working:
https://github.com/bluecatlabs/gateway-workflows/blob/518c6fe37d9bd86211a09ccdeb3b069eed061a09/Community/rest_api/dns_page.py#L687-L702

Here is a similar block of code for retrieving CName records that is working:
https://github.com/bluecatlabs/gateway-workflows/blob/518c6fe37d9bd86211a09ccdeb3b069eed061a09/Community/rest_api/dns_page.py#L593-L606

**Comment by vinny-sabatini** [comment]
We are no longer getting a 404 error, we are now getting the following error instead:

```
File "/portal/bluecat_portal/workflows/rest_api/dns_page.py", line 698, in get
     text_records = zone.get_children_of_type(zone.TextRecord)
 AttributeError: 'Zone' object has no attribute 'TextRecord'
```

**Comment by seanmalloy** [comment]
Fixed by #119 this bug can be close now.
